src/styles/blocks/header.scss

```
@@ -0,0 +1,58 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin-bottom: 50px;
```

```suggestion
  margin-bottom: 50px;
```

`header` is a block. You cannot give outer spacings (margins) for blocks. Either use padding or use a mix of classes, e.g. give the class name 'page-content' to your `body` tag, and `header` should have two classes 'header page-content__header'. Now `page-content__header` will be an element and you can give margins for this class